### PR TITLE
Preserve path in proxy kubeconfig if present in original kubeconfig

### LINKED ISSUE #855: Dashboard is not opening if cluster url includes path
**Describe the bug**
If Kubernetes cluster url includes path, for example Rancher is using url as `https://<host>/k8s/clusters/c-gfdpx`, Lens app cannot open dashboard.

### PR DESCRIPTION
**Pull request by nevalla**
Fixes #855 

Signed-off-by: Lauri Nevala <lauri.nevala@gmail.com>

### CODE REVIEW

File: src/main/context-handler.ts [hunk 1]
```diff
@@ -70,7 +70,8 @@ export class ContextHandler {
 
   async resolveAuthProxyUrl() {
     const proxyPort = await this.ensurePort();
-    return `http://127.0.0.1:${proxyPort}`;
+    const path = this.clusterUrl.path !== "/" ? this.clusterUrl.path : ""
+    return `http://127.0.0.1:${proxyPort}${path}`;
```
Comment (nevalla) [inline src/main/context-handler.ts:74]: `KubeconfigManager.createProxyKubeconfig` calls this method:
```
protected async createProxyKubeconfig(): Promise<string> {
   ...
  const proxyConfig: Partial<KubeConfig> = {
      currentContext: contextName,
      clusters: [
        {
          name: contextName,
          server: await contextHandler.resolveAuthProxyUrl(),
          skipTLSVerify: undefined,
        }
      ],
     ....
}
```

File: src/main/context-handler.ts [hunk 2]
```diff
@@ -70,7 +70,8 @@ export class ContextHandler {
 
   async resolveAuthProxyUrl() {
     const proxyPort = await this.ensurePort();
-    return `http://127.0.0.1:${proxyPort}`;
+    const path = this.clusterUrl.path !== "/" ? this.clusterUrl.path : ""
```
Comment (ixrock) [inline src/main/context-handler.ts:73]: Why we have to avoid `/` in the end?
In other words, maybe always add `this.clusterUrl.path` to the end?

**Review by ixrock — COMMENTED** [review]
LGTM

**Review by ixrock — APPROVED** [review]

File: src/main/context-handler.ts (hunk 2 above)
Comment (nevalla) [reply src/main/context-handler.ts:73]: `@kubernetes/client-node` does not like it

**Review by jakolehm — APPROVED** [review]
LGTM (and tested with Rancher 2.x)

File: src/main/context-handler.ts (hunk 2 above)
Comment (ixrock) [reply src/main/context-handler.ts:73]: Ok, thanks for info!

**Pull request merged**